pyexchange/uniswapv2_analytics.py

```diff
@@ -388,53 +410,38 @@ def get_all_trades(self, pair: str, page_number: int = 1) -> List[Trade]:
 
         trades_list = []
 
-        two_days_ago_unix = int(time.time() - (48 * 60 * 60))
-        start_time = two_days_ago_unix if two_days_ago_unix > self.all_last_pair_hour_timestamp else self.all_last_pair_hour_timestamp
+        # calculate starting block, assuming there's 15 seconds in a given block
+        current_block = self.get_current_block()
+        two_day_ago_block = int(current_block - (4 * 60 * 25))
```

Comment:
Assuming a block is 13.5 seconds, this block would be 22.5 hours ago.
But the variable name is `two_day_ago_block`; did you forget to multiply by 2 here?  Or should the variable name be adjusted?

Reply:
Yes, good point - I will update the variable names.